pagopa/io-app
# feat: [IOBP-930]  Remove payments redirect banner from wallet

## Short description
This pull request removes the banner that prompts users to go to the payments section when they want to pay a notice from the wallet screen.

## List of changes proposed in this pull request
- Delete `WalletPaymentsRedirectBanner` component and its redux variable

## How to test
- Navigate to the Portafoglio (Wallet) section.
- Verify that the banner prompting for payment redirection no longer appears.
